.github/ISSUE_TEMPLATE/author-bug-report.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Author bug report
+about: For contributors and developers
```

Comment:
By default, there will be an extra option (that, you should be warned from experience with other repos, is very easy to miss because it looks like a tip-and-trick text rather than a big green button) that presents no template at all.

My only reason for including this besides visibility was because then it’ll get the `bug` label pre-assigned. If you don’t think that’s worth it, I can drop it. But when you write

>I'm pretty sure contributors and developers know how to submit bug reports?

when I see a file only containing
```
<!-- This is for bugs involving the software used to build the docs.
If you are experiencing an issue viewing the docs, please use
another issue type. -->
```
which isn’t giving instructions or questioning how contributors and developers submit bug reports, so maybe we’re not looking at the same thing?